builtin/logical/ssh/path_roles.go

```diff
@@ -410,6 +431,7 @@ func (b *backend) pathRoleWrite(ctx context.Context, req *logical.Request, d *fr
 			KeyType:         KeyTypeOTP,
 			Port:            port,
 			AllowedUsers:    allowedUsers,
+			AlgorithmSigner: algorithmSigner,
```

Comment:
This setting is only meaningful for KeyTypeCA, so you probably shouldn't add it to the other types.

Reply:
Good point.  I should've made this a draft PR, it's not yet been tested.  I wanted to put something up to show the previous PR creator what was needed next to make this configurable.

Reply:
Fixed.